app/models/post.rb

```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :user
+
+  validates :post, presence: true
+end
```

Comment:
- **_There should be validations for `body` and `title` in the Post model ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/postValidation.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails)). This implies that you need to add the columns for the title and body (you can change the post column to body or not) to the post table_**